### PR TITLE
Open Graph: add image:width and image:height tags to fallback images

### LINKED ISSUE #2104: Open Graph: add og:image:width and og:image:height Open Graph tags
These 2 tags seem useful when pre-caching the image to be used in post previews:
https://developers.facebook.com/docs/sharing/best-practices#precaching

### PR DESCRIPTION
**Pull request by jeherve**
Part 2 of #2291

See #2283 for more discussion about this

Fixes #2104

### CODE REVIEW

**Comment by michaeldcain** [comment]
+1 this looks good to me

File: modules/theme-tools/site-logo/inc/functions.php [hunk 1]
```diff
@@ -31,6 +31,49 @@ function jetpack_get_site_logo( $show = 'url' ) {
...
+ *
+ * @return array $dimensions {
+ *		An array of dimensions of the Site Logo.
+ *
+ * 		@type string $width Width of the logo in pixels.
+ * 		@type string $height Height of the logo in pixels.
+ * }
+ */
+function jetpack_get_site_logo_dimensions() {
+	$dimensions = array();
```
Comment (dereksmart) [inline modules/theme-tools/site-logo/inc/functions.php]: No need to declare an empty array here

File: functions.opengraph.php [hunk 2]
```diff
@@ -237,12 +237,20 @@ function jetpack_og_get_image( $width = 200, $height = 200, $max_images = 4 ) {
 
 	// Second fall back, Site Logo
 	if ( empty( $image ) && ( function_exists( 'jetpack_has_site_logo' ) && jetpack_has_site_logo() ) ) {
-		$image['src'] = jetpack_get_site_logo( 'url' );
+		$image['src']        = jetpack_get_site_logo( 'url' );
+
+		$image_dimensions    = jetpack_get_site_logo_dimensions();
```
Comment (dereksmart) [inline functions.opengraph.php]: These three lines seem awkwardly spaced

File: modules/theme-tools/site-logo/inc/functions.php [hunk 3]
```diff
@@ -31,6 +31,49 @@ function jetpack_get_site_logo( $show = 'url' ) {
...
+	} else if ( 'thumbnail' == $size ) {
+		$dimensions  = array(
+			'width'  => get_option( 'thumbnail_size_w' ),
+			'height' => get_option( 'thumbnail_size_h' ),
+		);
+	} else {
+		global $_wp_additional_image_sizes;
+
+		$dimensions  = array(
+			'width'  => $_wp_additional_image_sizes[$size]['width'],
```
Comment (dereksmart) [inline modules/theme-tools/site-logo/inc/functions.php]: whitespace code style for variables should be like so: `[ $size ]`

**Comment by dereksmart** [comment]
couple super minor nitpicks, but otherwise this is all gravy

**Comment by jeherve** [comment]
All fixed, thank you!

**Pull request merged**